src/lv_objx/lv_label.h

```diff
@@ -164,6 +171,21 @@ static inline void lv_label_set_style(lv_obj_t * label, const lv_style_t * style
 {
     lv_obj_set_style(label, style);
 }
+
+/**
+ * @brief Set the selection start index.
+ * @param label pointer to a label object.
+ * @param index index to set. -1 to select nothing.
+ */
+void lv_label_set_selection_start( const lv_obj_t * label, int index );
```

Comment:
Please use `int32_t` here as well instead of `int`.

`const` seems misleading here because the label will be modified.

Reply:
fixed